### PR TITLE
Fix WS-NSB G6 post-merge review gaps

### PR DESCRIPTION
**Pull request by Bdavis1390**
## Summary

Corrects the post-merge review findings reported after PR #131:

- require at least one positive Hartmann case so magnetic damping and EM-dissipation checks cannot pass vacuously;
- compute temporal observed order from the actual effective time-step refinement ratios after final-time rounding;
- evaluate the steady Hartmann cosh ratio with an overflow-safe formulation at large Hartmann values;
- avoid overflow in the final Hartmann denominator by sequential scaling, preserving representable subnormal results beyond `Ha ≈ 1.34e154`.

Focused regression coverage includes a one-case sweep rejection, non-binary refinement ratio, `Ha=1000` cosh safety, and a nonzero `Ha=1e155` interior reference value.

## Exact-head evidence

Current head: `a5896385ed2ddf46f26eaaf219200ca19357fdf9`

All eight exact-head workflows pass, including **WS-NSB v1.1 G6 Gate** and **SARA Verified Local v1 Gate**.

The controlling exact-head Codex review reported no major issues on September 10, 2026. The corrected denominator-overflow P2 thread is dispositioned and resolved. This establishes review cleanliness only for the stated code scope and exact commit.

## Claims boundary

This remains **SIMULATED_ONLY** for the bounded 1D quasi-static Hartmann reference. It does not establish full MHD, plasma, laboratory, propulsion, shielding, or operational capability.

## Gate

**DRAFT / REVIEW CLEAN / BLOCK MERGE** pending explicit CRE1AWS incorporation authorization. No deployment or physical-capability promotion is authorized by this PR.

### CODE REVIEW

**Comment by Bdavis1390** [comment]
@codex review

Please review exact head `afb3abae88aa89cb6756365a13a9e8dee78eb9dc` against the three post-merge PR #131 findings: non-vacuous Hartmann sweep validation, effective-dt temporal-order calculation, and overflow-safe large-Hartmann steady reference. Also check the new regression tests and claims boundary.

File: deployments/sara_verified_local_v1/worldshepherd_sara/nsb_g6_transient_hartmann.py [hunk 1]
```diff
@@ -194,7 +194,17 @@ def steady_hartmann_exact_velocity(y: float, hartmann: float) -> float:
...
+        raise ValueError("steady Hartmann reference requires y in [-1, 1]")
+    if ha < 20.0:
+        cosh_ratio = math.cosh(ha * y) / math.cosh(ha)
+    else:
+        absolute_y = abs(y)
+        cosh_ratio = (
+            math.exp(ha * (absolute_y - 1.0))
+            + math.exp(-ha * (absolute_y + 1.0))
+        ) / (1.0 + math.exp(-2.0 * ha))
+    return (1.0 - cosh_ratio) / (ha * ha)
```
[the remark this answers is not in the document]
Comment (Bdavis1390) [reply deployments/sara_verified_local_v1/worldshepherd_sara/nsb_g6_transient_hartmann.py]: Fixed on exact head `a5896385ed2ddf46f26eaaf219200ca19357fdf9`: the denominator is now applied as sequential division, so `Ha=1e155` preserves the representable `1e-310` interior result. Regression coverage asserts it remains strictly nonzero. All eight exact-head workflows pass; requesting fresh review before disposition or merge.

**Comment by Bdavis1390** [comment]
@codex review

Please review exact head `a5896385ed2ddf46f26eaaf219200ca19357fdf9`, including the sequential large-Hartmann denominator scaling and `Ha=1e155` nonzero regression. All eight exact-head workflows pass. The PR remains draft and blocked from merge pending this review.

File: deployments/sara_verified_local_v1/worldshepherd_sara/nsb_g6_transient_hartmann.py [hunk 2]
```diff
@@ -194,7 +194,17 @@ def steady_hartmann_exact_velocity(y: float, hartmann: float) -> float:
     ha = abs(hartmann)
     if ha == 0.0:
         return 0.5 * (1.0 - y * y)
-    return (1.0 - math.cosh(ha * y) / math.cosh(ha)) / (ha * ha)
+    if abs(y) > 1.0:
+        raise ValueError("steady Hartmann reference requires y in [-1, 1]")
```
[the remark this answers is not in the document]
Comment (Bdavis1390) [reply deployments/sara_verified_local_v1/worldshepherd_sara/nsb_g6_transient_hartmann.py:198]: Confirmed after PR #133 merged. Corrected in draft PR #135, exact head `6d19c0dbc812200131635db2e1e8e1eab9bb93d0`: the bounded y-domain guard now executes before the zero-field early return, with a regression for `y=2, Ha=0`. PR #135 remains BLOCK MERGE pending CI and fresh review.

File: deployments/sara_verified_local_v1/worldshepherd_sara/nsb_g6_transient_hartmann.py [hunk 3]
```diff
@@ -431,8 +454,16 @@ def run_nsb_g6_benchmark(
     )
     temporal_errors = tuple(item.l2_velocity_error for item in temporal_cases)
     temporal_orders = (
-        _observed_order(temporal_errors[0], temporal_errors[1]),
-        _observed_order(temporal_errors[1], temporal_errors[2]),
+        _observed_order(
+            temporal_errors[0],
+            temporal_errors[1],
+            temporal_cases[0].effective_dt / temporal_cases[1].effective_dt,
```
[the remark this answers is not in the document]
Comment (Bdavis1390) [reply deployments/sara_verified_local_v1/worldshepherd_sara/nsb_g6_transient_hartmann.py:460]: Confirmed after PR #133 merged. Corrected in draft PR #135, exact head `6d19c0dbc812200131635db2e1e8e1eab9bb93d0`: the convergence summary now serializes the three effective time steps used to compute the reported orders, with a non-binary-rounding regression. PR #135 remains BLOCK MERGE pending CI and fresh review.
